addOns/encoder/src/main/javahelp/org/zaproxy/addon/encoder/resources/help/contents/encoder.html

```diff
@@ -0,0 +1,133 @@
+<code>&quot;2&#65504;&quot;</code> would be encoded as <code>\&quot;2\uFFE0\&quot;</code>.
+
+<H2>Decode tab fields</H2>
+This tab has the following fields:
+
+<H3>Base 64 Decode</H3>
+This read only field will be updated with the base 64 decoding of the text you enter.<br/>
+If there is no valid decoding then the field will be disabled.
+
+<H3>Base 64 URL Decode</H3>
+This read only field will be updated with the base 64 URL decoding of the text you enter. Base64URL is a modification to the primary base 64 standard 
+the purpose of which is the ability to use the encoding result as filename or URL address. Base64URL is described in 
+<a href="https://tools.ietf.org/html/rfc4648">RFC 4648</a>, which notes that the standard base 64 alphabet includes characters that are invalid for 
+URLs and file names (for example: plus sign and question mark are problematic). 
+If there is no valid decoding then the field will be disabled.
+
+<H3>URL Decode</H3>
+This read only field will be updated with the URL decoding of the text you enter.
+
+<H3>ASCII Hex Decode</H3>
+This read only field will be updated with the ASCII hex decoding of the text you enter.<br/>
+If there is no valid decoding then the field will be disabled.
+
+<H3>HTML Decode</H3>
+This read only field will be updated with the HTML decoding of the text you enter. For example, the text
+<code>&amp;quot;pi&amp;ntilde;ata&amp;quot;</code> would be decoded as <code>&quot;pi&ntilde;ata&quot;</code>.
+
+<H3>JavaScript Decode</H3>
+This read only field will be updated with the unescaped JavaScript literals of the text you enter. For example, the text
+<code>\&quot;2\uFFE0\&quot;</code> would be decoded as <code>&quot;2&#65504;&quot;</code>.
+
+<H2>Hash tab fields</H2>
+This tab has the following fields:
+
+<H3>SHA1 Hash</H3>
+This read only field will be updated with the SHA1 hash of the text you enter.
+
+<H3>MD5 Hash</H3>
+This read only field will be updated with the MD5 hash of the text you enter.
+
+<H2>Illegal UTF-8</H2>
+This tab has the following fields:
+
+<H3>2 byte</H3>
+This read only field will be updated with an illegal UTF-8 character sequence with 2 bytes.
+
+<H3>3 byte</H3>
+This read only field will be updated with an illegal UTF-8 character sequence with 3 bytes.
+
+<H3>4 byte</H3>
+This read only field will be updated with an illegal UTF-8 character sequence with 4 bytes.
+
+<H2>Unicode</H2>
+This tab has the following fields:
+
+<H3>Escaped Text</H3>
+This read only field will be updated with escaped Unicode characters. For example, the text
+<code>A&ccedil;ores</code> would be encoded as <code>%u0041%u00e7%u006f%u0072%u0065%u0073</code>.
+
+<H3>Unescaped Text</H3>
+This read only field will be updated with the unescaped Unicode characters. For example, the text
+<code>%u0041%u00e7%u006f%u0072%u0065%u0073</code> would be decoded as <code>A&ccedil;ores</code>.
+
+<H2>Accessed via</H2>
```

Comment:
We could add a generic entry, it's also possible to access it in other text components (e.g. Output tab, Script Console).